README.md

```diff
@@ -0,0 +1,40 @@
+# Freedom! Accounts Util
+A utility helper for OAuth2.0 calls to Freedom! Accounts that supports caching.
+
+## Installation:
+With NPM:
+```
+npm install freedom-accounts-util --save
+```
+
+With Yarn:
+```
+yarn add freedom-accounts-util
+```
+
+## Setup:
+```
+const accounts = require('freedom-accounts-util');
+
+accounts.configure({base_url: '', path: '', client_id: '', client_secret: '', client_expiry: 300, server_expiry: 300});
```

Comment:
Do we want to have an option for disabling cache?

This is usable in testing since we're mocking accounts. Since we already have caching of access token, it will not request again from accounts and the mock will not be consumed which can affect succeeding tests.
Although we can prevent it by cleaning the previous mocks, I think it's better to assume that the tests that request to API always have new access token (not in cache yet).

Reply:
Done:
00f031a